src/ixdat/techniques/reactor.py

```diff
@@ -277,17 +277,14 @@ class ReactorSpectroMeasurement(ReactorMeasurement, MSSpectroMeasurement):
     default_plotter = TPMSSpectroPlotter
 
 
-class ReactorCalibration(Calibration):
+class ReactorCalibration(Calculator):
     """A reactor calibration to calibrate inverse of meta_series"""
 
-    def __repr__(self):
-        # TODO: make __repr__'s consistent throught ixdat
-        return (
-            f"{self.__class__.__name__}"
-            f"(Calibration={self.name} for setup={self.setup} on date={self.date})"
-        )
+    # FIXME: Is this calculator really necessary?
```

Comment:
does it work similar enough to the MS part of ECMS?

Reply:
I mean the question is just whether every method/script that needs to take 1 divided something should just do the division itself.